DataConnectors/ForgeRock_CEF.json

```diff
@@ -0,0 +1,112 @@
+        }
+      }
+    ]
+  },
+  "instructionSteps" : [
+    {
+      "title" : "Configuration for the ForgeRock Common Audit (CAUD) for Microsoft Azure Sentinel",
+      "description" : "In ForgeRock, install and configure this Common Audit (CAUD) for Azure Sentinel per the documentation at https://github.com/javaservlets/SentinelAuditEventHandler. Next, in Azure, follow the below CEF steps."
+    },
+    {
+      "title" : "\n\n\n1. Linux Syslog agent configuration",
+      "description" : "Install and configure the Linux agent to collect your Common Event Format (CEF) Syslog messages and forward them to Azure Sentinel.\n\n> Notice that the data from all regions will be stored in the selected workspace",
+      "innerSteps" : [
+        {
+          "title" : "1.1 Select or create a Linux machine",
+          "description" : "Select or create a Linux machine that Azure Sentinel will use as the proxy between your security solution and Azure Sentinel this machine can be on your on-prem environment, Azure or other clouds."
+        },
+        {
+          "title" : "1.2 Install the CEF collector on the Linux machine",
+          "description" : "Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector colects CEF messages on port 514 TCP.\n\n> 1. Make sure that you have Python on your machine using the following command: python –version.\n\n> 2. You must have elevated permissions (sudo) on your machine.",
+          "instructions" : [
+            {
+              "parameters" : {
+                "fillWith" : [
+                  "WorkspaceId",
+                  "PrimaryKey"
+                ],
+                "label" : "Run the following command to install and apply the CEF collector:",
+                "value" : "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+              },
+              "type" : "CopyableLabel"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title" : "2. Forward Common Event Format (CEF) logs to Syslog agent",
+      "description" : "Set your security solution to send Syslog messages in CEF format to the proxy machine. Make sure you to send the logs to port 514 TCP on the machine’s IP address."
+    },
+    {
+      "title" : "3. Validate connection",
+      "description" : "Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python –version\n\n>2. You must have elevated permissions (sudo) on your machine",
```

Comment:
Please remove the weird characters after "python âversion" and replace with hyphen. You should be able to validate and see if this is fixed by the test utility https://portal.azure.com/?feature.BringYourOwnConnector=true per the data connector guidance. Please test and fix these.